### ISSUE TITLE
Ajax

### ISSUE THREAD

**Issue by vkabachenko** [ricco381/yii2-ticket#3]
Хорошо бы в пользовательской части обеспечить возможность вывода тикета в модальном окне. Для этого придется немного переделать представления, дописав там ajax-вызовы действий контроллеров. Это не проблема, пользователю вашего модуля все равно переписывать представления придется, хотя бы для изменения дизайна. Но хотелось бы не переписывать контроллеры. Думаю, достаточно немного изменить исходный код  TicketController: перед `return $this->redirect(...)` вставить `if (\Yii::$app->request->isAjax) return 'OK'`

**Comment by ricco381** [comment]
Вы можете создать fork и сделать этот функционал, я буду только рад этому, так как идея и вправду хорошая!

**Comment by vkabachenko** [comment]
ок, я сейчас занимаюсь интеграцией вашего модуля в проект. Как отладку закончу, напишу pull-request. Сейчас приходится свой контроллер использовать вместо вашего

**Comment by ricco381** [comment]
Спасибо!

**Comment by ricco381** [comment]
Прописав этот checkExtensionByMimeType параметр в значение false вы сделали из загрузки фалов огромную дыру для злоумышленников!

**Comment by vkabachenko** [comment]
Согласен. Но другого способа обойти ошибки при автоматическом определении mime-типа наверно нет- [https://github.com/yiisoft/yii2/issues/9888](url)  Может быть лучше прописать `'checkExtensionByMimeType' => $this->module->checkExtensionByMimeType,` и по умолчанию в модуле поставить `public $checkExtensionByMimeType = true`. Тогда модуль за возможные бреши отвечать не будет
